pkg/tuned/controller.go

```diff
@@ -158,16 +156,6 @@ type wqKey struct {
 	event string // object event type (add/update/delete) or pass the full object on delete
 }
 
-// Functions
```

Comment:
these are usually added to comply with the `flag` interface, not sure are actually unused

Reply:
the type `arrayFlags` was not used at all